bokeh/charts/builder.py

```diff
@@ -216,6 +216,10 @@ class Builder(HasProps):
         is (Column, Ascending).
         """)
 
+    source = Instance(ColumnDataSource)
+
+    hover = Either(List(Tuple(String, String)), List(String), Bool, default=None)
```

Comment:
I'm afraid calling it hover may create confusion with the `HoverTool` itself. What about calling it `tooltips` or `hover_tooltips` to not get confused?